_common/KeymanHosts.php

```diff
@@ -42,6 +42,24 @@ public static function Rebuild() {
       self::$instance = new KeymanHosts();
     }
 
+    /**
+     * Modifies $contents to regex the Keyman live hosts for Markdown files
+     * @param $contents
+     */
+    public function fixupHostReferences(&$contents) {
+      // Regex Keyman hosts
+      $contents = str_replace("https://s.keyman.com", $this->s_keyman_com, $contents);
+      $contents = str_replace("https://api.keyman.com", $this->api_keyman_com, $contents);
```

Comment:
Maybe we don't want to regex api.keyman.com because it breaks the build-test check for broken links
e.g. http://localhost:8888/keyboards/european, http://localhost:8888/keyboards/european2

Reply:
Those pages shouldn't have failed. Something else going on here.

Reply:
I re-ran the check and it failed only on gff-gez-7, which is now being fixed in #198.